### PR TITLE
Updated sensor disconnection message (CU-86dthh6wa)

### PR DESCRIPTION
**Pull request by Darrickg**
Changed the words "Ceiling Unit" to "Brave Sensor" for English and Spanish

Test Plan:

- [x] Passed all current test cases

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86dthh6wa Change "Ceiling Unit" in Sensor Disconnection Messages](https://app.clickup.com/t/86dthh6wa)

**Comment by Darrickg** [comment]
changed the messages in the google sheets for English, French and Spanish as well!

File: server/resources/translations/common.es_us.json [hunk 1]
```diff
@@ -15,8 +15,8 @@
...
+  "sensorDisconnectionInitial": "El Sensor Brave en {{ deviceDisplayName }} ({{ clientDisplayName }}) se ha desconectado.\nPresione el botón de reinicio dentro del orificio circular en el Sensor Brave con un alfiler. Si no recibe un mensaje de reconexión dentro de los próximos 15 a 20 minutos después de presionar el botón de reinicio, contáctenos enviando un correo electrónico a clientsupport@brave.coop.",
+  "sensorDisconnectionReminder": "El Sensor Brave en {{ deviceDisplayName }} ({{ clientDisplayName }}) sigue desconectado.\nPresione el botón de reinicio dentro del orificio circular en el Sensor Brave con un alfiler. Si no recibe un mensaje de reconexión dentro de los próximos 15 a 20 minutos después de presionar el botón de reinicio, contáctenos enviando un correo electrónico a clientsupport@brave.coop.",
```
Comment (micahdbak) [inline server/resources/translations/common.es_us.json:19]: Looks good!

**Review by micahdbak — COMMENTED** [review]
Make note of your changes in the CHANGELOG.md, please! And the respective docs for translations

**Comment by Darrickg** [comment]
added changelog, let me know if I can merge it :)

**Comment by micahdbak** [comment]
I'm gonna wait for @johnny-brave to take a look at this PR before I approve it, to make sure he's OK with the wording

**Review by johnny-brave — APPROVED** [review]
looks good to me!

**Review by micahdbak — APPROVED** [review]
Go for it @Darrickg :)

**Pull request merged**